checks/runtime.go

```diff
@@ -138,17 +137,17 @@ func CheckSensitiveDirs(client *client.Client) (res Result) {
 	return
 }
 
-func CheckSSHRunning(client *client.Client) (res Result) {
+func CheckSSHRunning(t actuary.Target) (res Result) {
 	var badContainers []string
 	res.Name = "5.6 Do not run ssh within containers"
-	containers := actuary.CreateContainerList(client)
+	containers := t.Containers
 
 	if !containers.Running() {
 		res.Skip("No running containers")
 		return
 	}
 	for _, container := range containers {
-		procs, _ := client.ContainerTop(container.ID, []string{})
+		procs, _ := t.Client.ContainerTop(container.ID, []string{})
```

Comment:
`t.Client` is a pointer to `client.Client` so it's basically the same as calling `client`